1.8/administration/installing/custom/cli.md

```diff
@@ -160,6 +160,11 @@ The DC/OS installation creates these folders:
     - 8.8.8.8
     ssh_port: 22
     ssh_user: <username>
+    use_proxy: 'true'
```

Comment:
@jsancio @cmaloney should i add a comment above line 163:
`# A custom proxy is optional. For details see the config documentation.`